cmd/kube-apiserver/app/server.go

```
@@ -227,6 +227,10 @@ func (s *APIServer) verifyClusterIPFlags() {
 	if s.ServiceClusterIPRange.IP == nil {
 		glog.Fatal("No --service-cluster-ip-range specified")
 	}
+	var ones, bits = s.ServiceClusterIPRange.Mask.Size()
+	if bits-ones > 20 {
```

This allows /12: 32 - 12 = 20 not > 20, so the error is not reported.